src/app/backend/resource/secret/secretdetail.go

```diff
@@ -36,9 +36,6 @@ type SecretDetail struct {
 
 	// Used to facilitate programmatic handling of secret data.
 	Type api.SecretType `json:"type"`
-
-	// Determines if the secret is hidden type in case of opaque and basicAuth types.
-	IsHiddenType bool `json:"isHiddenType"`
```

Comment:
We now just hide them all? When I looked at the default token I now can't see the 'namespace' anymore in a useful way